WalletWasabi.Fluent/ViewModels/Wallets/Send/OptimisePrivacyViewModel.cs

```diff
@@ -54,6 +54,11 @@ public partial class OptimisePrivacyViewModel : RoutableViewModel
 
 		private void OnNext(TransactionInfo transactionInfo)
 		{
+			if (_requestedTransaction == SelectedPrivacySuggestion!.TransactionResult)
+			{
+				_transactionInfo.UserDidntRequestOptimisation = true;
+			}
```

Comment:
Simplify? Your solution might be better to understand, it is up to you.

```
_transactionInfo.UserDidntRequestOptimisation = _requestedTransaction == SelectedPrivacySuggestion!.TransactionResult;
```

Reply:
done.